hs_core/models.py

```diff
@@ -1379,12 +1379,12 @@ class MyResourceContentType(pages.Page, hs_core.AbstractResource):
     content_object = generic.GenericForeignKey('content_type', 'object_id')
 
     def delete(self, using=None):
+        from hydroshare import hs_bagit
```

Comment:
Is there a reason this is not at the top of the file? (e.g. to avoid circular imports)  If not, this should be at the top of the file.